webui/react/src/hooks/useModal/UserSettings/useModalPasswordChange.tsx

```diff
@@ -41,7 +41,6 @@ const ModalForm: React.FC<Props> = ({ form, username = '' }) => (
     <Form.Item
       label={OLD_PASSWORD_LABEL}
       name={OLD_PASSWORD_NAME}
-      required
```

Comment:
we dont want this `required` because new user's password is empty and we have `validator`